tools/acrn-manager/README.rst

```diff
@@ -1,98 +1,99 @@
-``acrnctl``
-===========
+.. _acrnctl:
 
+acrnctl
+#######
 
-DESCRIPTION
-___________
 
-``acrnctl``: The ``acrnctl`` tool can help user create, delete, launch and stop UOSs.
-It runs under Service OS, and UOSs should be based on ``acrn-dm`` 
+Description
+***********
 
+The ``acrnctl`` tool helps users create, delete, launch, and stop a User
+OS (UOS).  The tool runs under the Service OS, and UOSs should be based
+on ``acrn-dm``.
 
-USAGE
-_____
 
-To see what it can do, just run:
 
-::
+Usage
+*****
 
-# acrnctl
+You can see the available ``acrnctl`` commands by running:
 
+.. code-block:: none
 
-or
+   # acrnctl help
+   support:
+     list
+     start
+     stop
+     del
+     add
+   Use acrnctl [cmd] help for details
 
-::
+Here are some usage examples:
 
-# acrnctl help
+Add a VM
+========
 
-you may see:
+The ``add`` command lets you add a VM by specifying a
+script that will launch a UOS, for example ``launch_UOS.sh``:
 
-::
+.. code-block:: none
 
-  support:
-    list
-    start
-    stop
-    del
-    add
-  Use acrnctl [cmd] help for details
+   # acrnctl add launch_UOS.sh -U 1
+   vm1-14:59:30 added
 
-There are examples:
+Note that the launch script must only launch one UOS instance.
+The VM name is important. ``acrnctl`` searches VMs by their
+names so duplicate VM names are not allowed. If the
+launch script changes the VM name at launch time, ``acrnctl``
+will not recognize it.
 
-(1) add a VM
-    Each time you can just add one VM. Suppose you have an UOS
-    launch script, such as launch_UOS.sh
+Delete VMs
+==========
 
-    you can run:
+Use the ``delete`` command with a VM name to delete that VM:
 
-::
+.. code-block:: none
 
-  # acrnctl add launch_UOS.sh -U 1
-    vm1-14:59:30 added
+   # acrnctl del vm1-14:59:30
 
-Note that, launch script shoud be able to launch ONE UOS. If
-it fail, it is better to print some error logs, to tell user
-the reason, so that he knows how to solve it.
-The vmname is important, the acrnctl searchs VMs by their
-names. so duplicated VM names are not allowed. Beside, if the
-launch script changes VM name at launch time, acrnctl will
-not recgonize it.
+List VMs
+========
 
-(2) delete VMs
+Use the ``list`` command to display VMs and their state:
 
-::
+.. code-block:: none
 
-  # acrnctl del vm1-14:59:30
+   # acrnctl list
+   vm1-14:59:30            untracked
+   vm-yocto                stop
```

Comment:
I'm pretty sure that the current output shows `stopped` (instead of `stop`). See https://github.com/projectacrn/acrn-hypervisor/blob/master/tools/acrn-manager/acrnctl.c#L107